.github/actions/build-keycloak/action.yml

```diff
@@ -35,7 +35,7 @@ runs:
         # Ensure this plugin is built first to avoid warnings in the build
         ./mvnw install -Pdistribution -am -pl distribution/maven-plugins/licenses-processor
         # By using "dependency:resolve", it will download all dependencies used in later stages for running the tests
-        ./mvnw install dependency:resolve -V -e -DskipTests -DskipExamples -DexcludeGroupIds=org.keycloak -Dsilent=true
+        ./mvnw install dependency:resolve -V -e -DskipTests -DskipExamples -DexcludeGroupIds=org.keycloak -Dsilent=true -DcommitProtoLockChanges=true
```

Comment:
I see the `proto.lock` file is created every time the build is executed at the moment. Should we change this to add `-DcommitProtoLockChanges=true` only if `github.event_name == 'pull_request' && startsWith(github.event.pull_request.base.ref, 'release/')`? We could add a new input for this action that would control addition of the parameter. WDYT?

Reply:
I set `-DcommitProtoLockChanges=true` for all builds in order to simplify the action. As this only results in a single file being written, IMO the potential milliseconds saved is not worth the added complexity that would be introduced in the action.

Reply:
We discussed offline and we agreed it is ok as is.